content/200-concepts/100-components/02-prisma-client/037-relation-queries.mdx

```diff
@@ -347,7 +347,7 @@ Please either use `include` or `select`, but not both at the same time.
 
 Instead, use nested `select` options:
 
-```ts
+```ts highlight=5,8-9;normal
```

Comment:
```suggestion
```ts highlight=5,8-12;normal
```
highlight full nested?

Reply:
This was difficult to decide — whether to highlight the entire relation query or only the bits that change... esp for queries that have already been mentioned/ highlighted in previous examples.